### ISSUE TITLE
Missing git tags

### ISSUE THREAD

**Issue by sol** [yesodweb/persistent#1109]
Git tags for several releases are missing (e.g. `2.5.10.1`).  This is tedious, when you e.g. have to apply patches to some specific version.

**Comment by MaxGabriel** [comment]
Agreed this isn't good. I don't have a particularly good enduring solution to this, though. Some sort of alias like:

```
$ alias upload
alias upload='mega-sdist --gittag . && stack upload tarballs/* && git push --tags'
```

(taken from https://github.com/snoyberg/mega-sdist#readme) might help

**Comment by parsonsmatt** [comment]
My bad. I'd be happy to have an automated solution for making releases like that.
